src/platform/android/java/chip/platform/NsdManagerServiceResolver.java

```diff
@@ -39,6 +41,7 @@ public class NsdManagerServiceResolver implements ServiceResolver {
   // discover TVs" is fixed
   // private static final long RESOLVE_SERVICE_TIMEOUT = 30000;
   private static final long RESOLVE_SERVICE_TIMEOUT = 5000;
+  private static final long BROWSE_SERVICE_TIMEOUT = 5000;
```

Comment:
```suggestion
  private static final long RESOLVE_SERVICE_TIMEOUT_SEC = 5000L;
  private static final long BROWSE_SERVICE_TIMEOUT_SEC = 5000L;
```

Reply:
Done for the new constant I introduced in the now separate file.